website/docs/r/service_dictionary_items_v1.html.markdown

```diff
@@ -0,0 +1,156 @@
+---
+layout: "fastly"
+page_title: "Fastly: service_dictionary_items_v1"
+sidebar_current: "docs-fastly-resource-service-v1"
+description: |-
+  Provides a grouping of fastly dictionary items that can be applied to a service. 
+---
+
+# fastly_service_dictionary_items_v1
```

Comment:
It would be good to add a comment in this documentation regarding the use of lifecycle ignore_changes if the user doesn't want TF to manage item state after initial apply.

Reply:
I have expanded the description to cover this and provided an example of the lifecycle ignore_changes with the resource.

Reply:
documentation has been expanded and will need a more specific review.